modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -3262,15 +3267,17 @@ AUI.add(
 
 						var formNode = instance.get('formNode');
 
-						if (event.form.attr('name') === formNode.attr('name')) {
+						if ((event.form.attr('name') === formNode.attr('name')) && instance.get('synchronousFormSubmission')) {
 							instance.updateDDMFormInputValue();
 						}
 					},
 
 					_onSubmitForm: function(event) {
 						var instance = this;
 
-						instance.updateDDMFormInputValue();
+						if (instance.get('synchronousFormSubmission')) {
```

Comment:
@antonio-ortega , wouldn't it be better to move this condition checking to the [event handlers assignment](https://github.com/rafaprax/liferay-portal/blob/926e816765069a3835e28fe7cadace11fe50a3be/modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js#L3017-L3019) to avoid calling this method unnecessarily?